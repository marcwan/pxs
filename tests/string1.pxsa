MODULE_BODY:
        ADDVAR          x
        ADD             "cat's ", "are cute", x
        DEBUGPRINT      x
LABEL1:
        EXIT            0
END_MODULE_BODY
