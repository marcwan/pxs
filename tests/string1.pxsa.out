DEBUGPRINT: x = cat's are cute
